src/watchmaker/__init__.py

```diff
@@ -52,6 +51,9 @@ def __init__(self, arguments):
             log_dir (str) or log_file (str):
                 Path to log directory or file for logging.
         """
+        self.log = logging.getLogger(
+            '{0}.{1}'.format(__name__, self.__class__.__name__)
+        )
```

Comment:
Is this one necessary if you have it in the __init__ function of WatchmakerException?